app/javascript/packs/webauthn-setup.ts

```diff
@@ -52,19 +47,6 @@ function webauthn() {
       })
       .catch((err) => reloadWithError(err.name, { force: true }));
   });
-  const input = document.getElementById('nickname') as HTMLInputElement;
-  input.addEventListener('keypress', function (event) {
-    if (event.keyCode === 13) {
-      // prevent form submit
-      event.preventDefault();
-    }
-  });
-  input.addEventListener('keyup', function (event) {
-    event.preventDefault();
-    if (event.keyCode === 13 && input.value) {
-      continueButton.click();
-    }
-  });
```

Comment:
Really glad we can change this to use native submit behaviors 👍

Reply:
yup good catch, this feels a lot cleaner.